packages/@divvi/mobile/src/walletConnect/constants.ts

```diff
@@ -65,6 +68,11 @@ export function getDisplayTextFromAction(
       title: t('walletConnectRequest.signPayloadTitle'),
       action: t('allow'),
     },
+    [SupportedActions.wallet_getCapabilities]: {
+      description: t('walletConnectRequest.getCapabilities', { dappName }),
+      title: t('walletConnectRequest.getCapabilitiesTitle'),
+      action: t('allow'),
+    },
```

Comment:
I'd say we don't need consent from the user for that. If they connected to the dapp it's enough. In that case we can skip the UI.
MetaMask or Ambire don't ask consent for this either.

I guess maybe the concern is the app being redirected to and then back directly without explaining anything to the user?